addons/web/static/src/views/fields/properties/property_value.xml

```diff
@@ -40,8 +40,8 @@
                 <input
                     t-att-id="props.id"
                     class="o_input o_field_property_input"
-                    type="number"
-                    t-att-value="propertyValue"
+                    type="text"
+                    t-att-value="displayValue"
```

Comment:
Should we make it "displayValue" for the integer field as well?
As "formatInteger" can do stuff with humanReadable and thousand separators, seems like it could be an issue too.